CONTRIBUTING.md

```diff
@@ -75,7 +75,6 @@ The Iter8 project consists of the following repos.
 1. [iter8-tools/iter8](https://github.com/iter8-tools/iter8): source for the Iter8 CLI
```

Comment:
`hub` goes away. And description of Iter8 repo becomes, source code for Iter8 CLI, and Helm charts.

Reply:
Further, 

http://localhost:8000/latest/contributing/#build-and-push-iter8-image -- we just removed `Dockerfile.dev`. So, perhaps, we can introduce a bash heredoc command to create this file?

```shell
cat << EOF > Dockerfile.dev
the contents of the previous Dockerfile.dev go here ... 
EOF
```